core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -3084,4 +3084,8 @@ public String getKerberosJaasConfPath() {
     public String getKerberosPrincipal() {
         return getOptional("kylin.kerberos.principal");
     }
+
+    public boolean isReuseParentDataSource() {
+        return Boolean.parseBoolean(getOptional("kylin.engine.spark.reuse.parent.rdd", FALSE));
+    }
```

Comment:
IMO, don't need this configuration, we can add a configuration named 'kylin.engine.spark.parent-dataset.storage.level' and set the storage level directly, if the storage level is 'NONE', it means it doesn't need to cache parent dataset.

Reply:
thanks,  good advice indeed ~